src/services/mondomarket/index.jsx

```
@@ -0,0 +1,238 @@
+import {
+  bookMarkItem,
+  removeBookMarkItem,
+  sendOffer,
+} from "../../app/features/mondomarket";
+import { useState } from "react";
+import AppInput from "../../components/form/AppInput";
+import Button from "../../components/form/Button";
+import { Spinner } from "../../components/theme/Loader";
+
+export const MondoDetailsViewer = ({
+  mondo,
```

ESLint

[error] 'mondo' is missing in props validation

(react/prop-types)